cmd/monaco/integrationtest/v2/settings_integration_test.go

```diff
@@ -344,6 +350,12 @@ func TestOrdered_InsertAtFrontWorksWithoutBeingSet(t *testing.T) {
 	})
 }
 
+func filterObjectsForScope(pgiMeId string) func(object dtclient.DownloadSettingsObject) bool {
```

Comment:
nit: Should this also move to towards the end of the file with the other helper functions?

Reply:
Moved all (also the other) functions down